### ISSUE TITLE
[$250][HOLD https://github.com/Expensify/App/pull/52164] Add workspace creation confirmation to workspace switcher.

### ISSUE THREAD

**Issue by danielrvidal** [Expensify/App#52075]
We're already [adding the workspace confirmation to Global Create and in Settings > Workspaces](https://github.com/Expensify/App/issues/51504). We forgot to also put the confirmation in another flow, the workspace switcher. 

Can you please add the confirmation in the following screens as well. 

Workspace switcher > Get started:
![image](https://github.com/user-attachments/assets/edcdf3a4-f843-4929-a80b-5140045aa321)

Workspace switcher > + New Workspace:
![image](https://github.com/user-attachments/assets/b7a80f17-c3ce-45b5-a8b7-cea8d355db9a)

Video of me showing these screens: 
https://github.com/user-attachments/assets/81f82325-a96b-43da-ac7a-560a277ec2eb

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~021853885696160356282</li>
        <li>Upwork Job ID: 1853885696160356282</li>
        <li>Last Price Increase: 2024-11-05</li>
<li>Automatic offers: </li>
<ul>
<li>allgandalf | Reviewer | 104757767</li>
<li>Krishna2323 | Contributor | 104757770</li>
</ul></li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @</details>

**Comment by rafecolton** [comment]
Is there a WIP PR for this?

**Comment by rafecolton** [comment]
Done

**Comment by danielrvidal** [comment]
Still on hold. The other issue is making daily progress.

**Comment by rafecolton** [comment]
Still on hold

**Comment by rafecolton** [comment]
I'm moving this to a weekly so it doesn't ping so often. We can move it back once it's off hold.

**Comment by rafecolton** [comment]
Lol @allgandalf I didn't see your message 🙈 same idea 🧠

**Comment by danielrvidal** [comment]
We're still blocked on this one. Hopefully @allgandalf will be done with this soon: https://github.com/Expensify/App/pull/53845

**Comment by danielrvidal** [comment]
Sorry for the delay on this, I just added the BZ assignment who can help with payment.

**Comment by trjExpensify** [comment]
>@danielrvidal @allgandalf, I'm not sure when, but I made the necessary changes to update this in the https://github.com/Expensify/App/pull/52164, and it's working.

In which case, payment will be handled in https://github.com/Expensify/App/issues/51504 which that PR is linked to, right?

**Comment by trjExpensify** [comment]
Yeah, looks like a two line change in [that PR](https://github.com/Expensify/App/pull/52164/files) to show the same modal on the `WorkspaceSwitcherPage`:

<img width="1157" alt="Image" src="https://github.com/user-attachments/assets/771ea421-ceb8-4dff-8ae9-9ad2175a05e8" />

Somewhat reflective of forgetting this update was made in the original PR.
